### DISCUSSION TITLE
Vimeo Download Issues. Downloads stop prematurely.

### DISCUSSION THREAD

**Discussion by xwasinx** [aclap-dev/video-downloadhelper#1015] [Help and Q&A]
### Your question

Hi,
I'm reaching out for help with an issue I've been having with VDH in the last days. Despite trying various solutions, I'm still unable to download videos from Vimeo without encountering problems.
The issue is that when I try to download any video from Vimeo using the VDH extension, the downloads stop prematurely. I've tried downloading videos multiple times, but they only complete when I retry the process several times. This issue persists across different computers and browsers (Brave and Chrome) on Windows 10.
I've checked the VDH extension settings and they are configured by default. I've also tried reinstalling the extension, but the problem remains.
Has anyone else experienced similar issues with the VDH extension? Are there any known solutions or workarounds that I might be missing? Any help or advice would be greatly appreciated.
Thank you so much for your help.

### Addon & CoApp version and details

version: 9.0.2.12
target: google
channel: stable
lang: es-ES
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"8ec52c44ec77491c"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "concurrent_downloads_max": 6,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": false,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 9,
    "successfull_dl": 53,
    "never_show_successfull_dl_message": true
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3880, Firefox 128.0 64-bit, licensed VDH 9.0.2.14a1 beta, CoApp 2.0.19.

I went to vimeo.com/watch & picked the first thing on the page (which makes it essentially a random pick).  That was https://vimeo.com/973212054.  VDH downloaded the whole thing just fine & it played just fine, video & audio, all the way to the end, including the closing credits.

Vimeo used to be a huge problem for VDH.  But that was back in the VDH 7 & early VDH 8 days.  Since Paul upgraded the CoApp to use ffmpeg for downloading, the problems all went away.  Vimeo uses its own quirky data structures but Paul has adapted VDH to interpret them properly.  He has not yet implemented support for captions.  This particular video does have English captions.  They are apparently machine generated but they seemed reasonable.  Slightly out of synch here & there, a word or two missing or not quite right, but mostly OK.

However, I'm talking about Firefox.  It's quite possible that things don't work entirely as they should on the other browsers.  This seems to be a common pattern.  Things work great on Firefox but not so great on the other browsers.  I don't use the other browsers so I don't know what the story is there.  All I can tell you is that Vimeo presents no problem for VDH on Firefox.  I hope that helps @paulrouget Paul narrow things down.

**Comment by xwasinx** [comment]
UPDATE: Over the past couple of days, I've been having some trouble downloading videos from Vimeo. I would often have to retry the download 5 or more times before it would finally work. However, after posting my message in the forum, I decided to try downloading again and was able to successfully download over 30 videos without any issues.
It seems this was likely a temporary problem on the Vimeo server side. After posting my initial message, the downloads started working smoothly again. This was probably just an isolated incident and not an ongoing problem with Vimeo and the Video DownloadHelper (VDH) tool.
Thank you GrampaWillWilly for your help
